client/environment_import_test.go

```diff
@@ -0,0 +1,134 @@
+package client_test
+
+import (
+	. "github.com/env0/terraform-provider-env0/client"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"go.uber.org/mock/gomock"
+)
+
+var _ = Describe("Environment Import Client", func() {
+	mockEnvironmentImport := EnvironmentImport{
+		Id:         "id",
+		Name:       "name",
+		IacType:    "tofu",
+		IacVersion: "1.0",
+		Workspace:  "workspace",
+		Variables: []Variable{{
+			Name:        "name",
+			Value:       "value",
+			IsSensitive: false,
+			Type:        "string",
+		},
+		},
+	}
+
+	Describe("EnvironmentImportGet", func() {
+		var result *EnvironmentImport
+
+		BeforeEach(func() {
+			mockHttpClient.EXPECT().
+				Get("/staging-environments/"+mockEnvironmentImport.Id, nil, gomock.Any()).
+				Do(func(path string, request interface{}, response *EnvironmentImport) {
+					*response = mockEnvironmentImport
+				})
+			result, _ = apiClient.EnvironmentImportGet(mockEnvironmentImport.Id)
+		})
+
+		It("Should get organization id", func() {
+			organizationIdCall.Times(1)
+		})
+
+		It("Should return GpgKeys", func() {
+			Expect(*result).To(Equal(mockEnvironmentImport))
+		})
+	})
+
+	Describe("EnvironmentImportCreate", func() {
+		var result *EnvironmentImport
+		var err error
+
+		BeforeEach(func() {
+			mockOrganizationIdCall(organizationId)
+
+			payload := EnvironmentImportCreatePayload{
+				Name:       "name",
+				IacType:    "tofu",
+				IacVersion: "1.0",
+				Workspace:  "workspace",
+				Variables: []Variable{{
+					Name:        "name",
+					Value:       "value",
+					IsSensitive: false,
+					Type:        "string",
+				},
+				}}
+
+			expectedPayload := struct {
+				OrganizationId string `json:"organizationId"`
+				EnvironmentImportCreatePayload
+			}{
+				organizationId,
+				payload,
+			}
+
+			httpCall = mockHttpClient.EXPECT().
+				Post("/staging-environments", expectedPayload, gomock.Any()).
+				Do(func(path string, request interface{}, response *EnvironmentImport) {
+					*response = mockEnvironmentImport
+				})
+
+			result, err = apiClient.EnvironmentImportCreate(&payload)
+		})
+
+		It("Should get organization id", func() {
+			organizationIdCall.Times(1)
```

Comment:
move the times(1) to the BeforeEach.
This It isn't needed.

Reply:
is it important to even test this is called? if the mock is missing the test fails

Reply:
[a68bee0](https://github.com/env0/terraform-provider-env0/pull/855/commits/a68bee0b4fa0d69bfd3180df2e0e2c21d84a0dfc)